btr-web/btr-main-app/pages/ownerChange.vue

```diff
@@ -81,14 +81,19 @@ const showSignificantDateError = ref(false)
 const showAddIndividualError = ref(false)
 
 const significantIndividualChangeDate = (event: Date) => {
-  currentSIFiling.value.effectiveDate = dateToString(event, 'YYYY-MM-DD')
+  const effectiveDate = dateToString(event, 'YYYY-MM-DD')
+  currentSIFiling.value.effectiveDate = effectiveDate
+  for (const si of currentSIFiling.value.significantIndividuals) {
+    si.startDate = effectiveDate
```

Comment:
Hey hrvoje I think this will overwrite all the existing SI start dates to the current one so it will reintroduce the old bug? This should only update newly added SIs -- I think you can just check the `si.action == FilingActionE.ADD`. Can you add a test for this too as its one of the more confusing flows for us

Reply:
Yeah, check was there, but rebase somehow removed it o.O.

Reply:
awesome thanks